.github/workflows/build-test-lint-publish.yml

```diff
@@ -104,12 +104,41 @@ jobs:
           npm ci --legacy-peer-deps
           npm run build
 
-      - name: Publish the package
+      - name: Publish the package to npm
         id: publish
         uses: JS-DevTools/npm-publish@v1
         with:
           token: ${{ env.NPM_TOKEN }}
-
-      - if: steps.publish.outputs.type != 'none'
-        run: |
-          echo "Version changed: ${{ steps.publish.outputs.old-version }} => ${{ steps.publish.outputs.version }}"
+          # only publish if the version in package.json has changed
+          check-version: true
+
+      - name: Add git tag ${{ steps.publish.outputs.version }}
+        if: steps.publish.outputs.type != 'none'
+        uses: actions/github-script@v5
+        env:
+          NEW_VERSION: ${{ steps.publish.outputs.version }}
+          OLD_VERSION: ${{ steps.publish.outputs.old-version }}
+        with:
+          script: |
+            console.log(`Version changed: ${OLD_VERSION} => ${NEW_VERSION}`);
```

Comment:
question: where does this log to?

Reply:
I believe it will output into the GitHub Action workflow log.  I'm not 100% sure, so once this is merged I'll be verifying that it does.

Reply:
i'm pretty sure this will just log to stdout as you said